test/e2e/tests/multichain/asset-list.spec.ts

```diff
@@ -28,9 +28,12 @@ function buildFixtures(title: string, chainId: number = 1337) {
 }
 
 describe('Multichain Asset List', function (this: Suite) {
-  if (!process.env.PORTFOLIO_VIEW) {
-    return;
-  }
+  // Apply to all tests in this suite
+  before(function () {
+    if (!process.env.PORTFOLIO_VIEW) {
+      this.skip();
+    }
+  });
```

Comment:
`return` was a dirty way to do it, `skip()` tells our reporter that it was skipped

(Note, there's still an issue if **every** test in a TestFile is skipped, like right here.  We'll have to fix that later.)